ansible/deploy_load_test.yml

```diff
@@ -20,3 +20,15 @@
     - include: roles/load_test/tasks/install_host.yml
   roles:
     - {role: load_test, tags: 'load_test'}
+
+- name: Prototype service
+  hosts:
+    - load_test_host
```

Comment:
@NoahCarnahan wasn't sure which machine to deploy this on but picked this one since it required less setup.